addons/api/package.json

```diff
@@ -28,7 +28,7 @@
     "ember-cli-htmlbars": "^6.1.0",
     "ember-cli-mirage": "^2.4.0",
     "ember-copy": "^2.0.1",
-    "ember-data": "^3.28.12",
+    "ember-data": "~4.4.0",
```

Comment:
Changed our semver usage to `~` to match `ember-source` and we stay in step with the minor versions